src/include/ceph_features.h

```diff
@@ -84,6 +84,7 @@
 // duplicated since it was introduced at the same time as CEPH_FEATURE_CRUSH_TUNABLES5
 #define CEPH_FEATURE_NEW_OSDOPREPLY_ENCODING (1ULL<<58) /* New, v7 encoding */
 #define CEPH_FEATURE_FS_FILE_LAYOUT_V2       (1ULL<<58) /* file_layout_t */
+#define CEPH_FEATURE_FS_BTIME                (1ULL<<59) /* btime */
```

Comment:
I'm fine with either combining this flag with other features added at the same time or indiciating it in some other way. Just point me in the right direction for it...